packages/dds/tree/src/feature-libraries/sequence-field/invert.ts

```diff
@@ -32,29 +26,32 @@ export function invert<TNodeChange>(
     invertChild: NodeChangeInverter<TNodeChange>,
 ): Changeset<TNodeChange> {
     // TODO: support the input change being a squash
-    const opIdToTag = (id: OpId): ChangesetTag => {
-        return DUMMY_INVERT_TAG;
+    const opIdToTag = (id: OpId): RevisionTag | undefined => {
```

Comment:
What is this function supposed to do? Do we still plan to use `OpId`s for something?

Reply:
This is left over from the time when I started putting in place the support for a single changeset to have marks from different revision tags. I could simplify the code to pass the `RevisionTag` through directly. I just went for the smallest change but I'm happy to clean that up as part of this PR if you think that's worth doing here.

Reply:
Either way is fine with me.

Reply:
Leaving it as is since it doesn't really fit the scope the this PR.